packages/playground/remote/src/lib/offline-mode-cache.ts

```diff
@@ -61,6 +60,27 @@ export class OfflineModeCache {
 		await this.cache.addAll([...websiteUrls, ...['/']]);
 	}
 
+	/**
+	 * Remove outdated files from the cache.
+	 *
+	 * We cache data based on `buildVersion` which is updated whenever Playground
+	 * is built. So when a new version of Playground is deployed, the service
+	 * worker will remove the old cache and cache the new assets.
+	 *
+	 * If your build version doesn't change while developing locally check
+	 * `buildVersionPlugin` for more details on how it's generated.
+	 */
+	async purgeEverythingFromPreviousRelease() {
+		// @TODO: Ensure an older service worker won't ever remove the assets of a newer service worker,
+		//        even if this is accidentally called in the older worker.
+		const keys = await caches.keys();
+		const oldKeys = keys.filter(
+			(key) =>
+				key.startsWith(CACHE_NAME_PREFIX) && key !== LATEST_CACHE_NAME
+		);
+		return Promise.all(oldKeys.map((key) => caches.delete(key)));
```

Comment:
Is there a reason we aren't just using [CacheStorage.delete()](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage/delete)? Are there other things that may be cached that we want to respect?

Reply:
Oh such a good idea!

Reply:
>> Is there a reason we aren't just using CacheStorage.delete()? Are there other things that may be cached that we want to respect?
>
> Oh such a good idea!

😅 We already were using CacheStorage.delete() here, but I had misread. It was such a good idea that we were already doing it.